custom/conf/app.example.ini

```diff
@@ -1067,6 +1067,9 @@ ROUTER = console
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; List of reasons why a Pull Request or Issue can be locked
 ;LOCK_REASONS = Too heated,Off-topic,Resolved,Spam
+;; Maximum number of pinned Issues
+;; Set to 0 to disable pinning Issues
+;MAX_PINNED = 6
```

Comment:
wonder why limit max number?

Reply:
> wonder why limit max number?

At a guess UX concerns. 3 or 6 is a reasonable number (in rows of 3), anymore and humans will lose focus. 
Take this to the extreme and be able to pin inf... Then the point of pinning is lost while also inheriting a major UX issue